pandas/tests/groupby/test_rows.py

```diff
@@ -0,0 +1,302 @@
+""" Test GroupBy.rows positional grouped indexing GH#42864"""
+
+import random
+
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+
+
+@pytest.fixture()
+def small_df():
+    data = [
+        [0, "a", "a0_at_0"],
+        [1, "b", "b0_at_1"],
+        [2, "a", "a1_at_2"],
+        [3, "b", "b1_at_3"],
+        [4, "c", "c0_at_4"],
+        [5, "a", "a2_at_5"],
+        [6, "a", "a3_at_6"],
+        [7, "a", "a4_at_7"],
+    ]
+    df = pd.DataFrame(data, columns=["Index", "Category", "Value"])
+    return df.set_index("Index")
+
+
+@pytest.fixture()
```

Comment:
same as above (you may need to make a slightly longer name to be meaningful)

Reply:
Done